frontend_tests/node_tests/example7.js

```diff
@@ -50,14 +50,14 @@ const {run_test} = require("../zjsunit/test");
 */
 
 const channel = mock_esm("../../static/js/channel");
-const message_list = mock_esm("../../static/js/message_list");
 const message_lists = mock_esm("../../static/js/message_lists");
 const message_viewport = mock_esm("../../static/js/message_viewport");
 const notifications = mock_esm("../../static/js/notifications");
 const unread_ui = mock_esm("../../static/js/unread_ui");
 
-message_lists.current = {};
-message_lists.home = {};
+message_lists.current = {is_current_narrowed: () => true};
+message_lists.home = {is_current_narrowed: () => false};
```

Comment:
ERROR: frontend_tests/node_tests/example7.js no longer has complete node test coverage
  Lines missing coverage: 59

ERROR: frontend_tests/node_tests/message_events.js no longer has complete node test coverage
  Lines missing coverage: 17

Test are failing due to missing coverage in these lines, which is super weird to me. Any idea what am I doing wrong here?

Reply:
No, this is quite mysterious.